WebService/Runtime/Config.cs

```diff
@@ -277,7 +277,7 @@ private static IServicesConfig GetServicesConfig(IConfigData configData)
                 SimulationsStorage = GetStorageConfig(configData, SIMULATIONS_STORAGE_KEY),
                 DevicesStorage = GetStorageConfig(configData, DEVICES_STORAGE_KEY),
                 PartitionsStorage = GetStorageConfig(configData, PARTITIONS_STORAGE_KEY),
-                UserAgent = configData.GetString(USER_AGENT_KEY),
+                UserAgent = configData.GetString(USER_AGENT_KEY, "devicesimulation"),
                 StatisticsStorage = GetStorageConfig(configData, STATISTICS_STORAGE_KEY),
```

Comment:
please make this a constant and then use here

Reply:
Updated.